src/plugins/Trader/settings.ts

```diff
@@ -0,0 +1,11 @@
+import { createNetworkSettings } from '../../settings/createSettings'
+import { Platform } from './types'
+
+export const currentTrendingViewPlatformSettings = createNetworkSettings('currentTrendingViewPlatformSettings')
```

Comment:
Hmm I think plugin should not share the same namespace of Maskbook settings. It will become a conflict.